Metallum/Classes/View/BandCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  BandCell.swift
+//  Metallum
+//
+//  Created by AppleTree on 3/8/16.
+//  Copyright © 2016 OlhaF. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class BandCell: UITableViewCell {
```

Comment:
if it is subclass of UITableViewCell - should be BandTableViewCell
if it is subclass of UICollectionViewCell - should be BandCollectionViewCell